raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -35,6 +35,7 @@ contract NettingChannelContract {
 
         data.token = Token(token_address);
         data.settle_timeout = timeout;
+        data.opened = block.number;
```

Comment:
As discussed elsewhere, I went with replacing the `isopen()` function with a `can_transfer()` and declare the channel state `opened` as soon as the contract is created.